### DISCUSSION TITLE
Incorrect display of repositories

### DISCUSSION THREAD

**Discussion by andrey-helldar** [yhype/yhype-feedback#26] [Q&A] [answered]
I had a project that was located at https://github.com/andrey-helldar/cache. I subsequently moved it to the organization and it is now available at https://github.com/TheDragonCode/laravel-cache.

The project is active and used in applications.

But in the list it shows up as "andrey-helldar/cache" with the label "Archived".

This is despite the fact that I am looking at the list of projects by organization: https://yhype.me/github/accounts/thedragoncode.

And there are many such entries. I gave an example of one repository from them.

<details>
<summary>Screenshots</summary>

![image](https://github.com/yhype/yhype-feedback/assets/10347617/1e8e1544-1cca-4256-8d4f-e2950b99819c)
![image](https://github.com/yhype/yhype-feedback/assets/10347617/0b7213a5-9ddb-44d2-802e-bbbd4e632155)

</details>

In fact, all of these repositories have been moved to The Dragon Code organization.

For example, "publisher" has a status of "public archive" (https://github.com/TheDragonCode/publisher), but is listed as "private".

**Comment by antonkomarev** [comment, marked as answer]
@andrey-helldar Let's wait for the night and check it again tomorrow. At night there will be repositories cache re-sync. Cache re-syncing only for active users. Maybe it was cached earlier.

**Reply by andrey-helldar** [reply]
Got it. Standing by.

**Reply by andrey-helldar** [reply]
@antonkomarev, was updated. It's works! Thank you!

![image](https://github.com/yhype/yhype-feedback/assets/10347617/58ab437a-686f-4230-acd8-d74bec2d4898)

**Reply by antonkomarev** [reply]
Thank you for the feedback! Glad it works
